test/core/Safe.ModuleManager.spec.ts

```diff
@@ -4,6 +4,7 @@ import { AddressZero } from "@ethersproject/constants";
 import { getSafeWithOwners, getMock } from "../utils/setup";
 import { executeContractCallWithSigners } from "../../src/utils/execution";
 import { AddressOne } from "../../src/utils/constants";
+import crypto from "crypto";
```

Comment:
We can remove this by using `ethers.hexlify(ethers.randomBytes(32));`

Reply:
https://github.com/safe-global/safe-smart-account/pull/758/commits/110ad2f0e2a38181230f8ece49c094e2da193e1c